amazon-msk/cdk/secure-public-access/README.md

```diff
@@ -161,7 +182,16 @@ aws acm-pca list-certificate-authorities --query 'CertificateAuthorities[*].[Arn
 
 Note down the ARN of the ACM Private Certificate Authority you want to use.
 
-### Disable CloudWatch Integration
+### CloudWatch Integration
+
+```json
+    "cloudwatch":
+    {
+        "disable": false,
+        "logGroupName": "<your public tls certificate key ARN>",
+        "port": "<your public port>"
```

Comment:
Typos in <...> comments.

Perhaps we can go with `logGroup` instead of `logGroupName`?

Reply:
Perhaps even
```
"cloudwatch":
{
  "logs":
  {
    "group": ...
  },
  "metrics":
  {
    "namespace": ...
  }
}
```
?